server/server.go

```diff
@@ -107,6 +112,10 @@ func NewServer(ctx context.Context, prof Profile) (*Server, error) {
 	s := &Server{
 		profile:   prof,
 		startedTs: time.Now().Unix(),
+		errorRecordRing: api.ErrorRecordRing{
```

Comment:
If we are encapsulating this in a struct, then we can create a method NewErrorRecordRing() and put errorRecordCount = 100 in that file